demisto_sdk/commands/unify/generic_module_unifier.py

```diff
@@ -0,0 +1,115 @@
+
+import json
+import os
+import sys
+from argparse import FileType
+from typing import Dict, Optional
+
+import click
+
+from demisto_sdk.commands.common.constants import FileType
+from demisto_sdk.commands.common.tools import get_pack_name, print_error, find_type
+
+
+class GenericModuleUnifier:
+    """
+    Unifies a GenericModule object with it's Dashboards
+    """
+
+    def __init__(self, input: str, output: str = '', force: bool = False):
+        """
+        Init a GenericModuleUnifier
+        Args:
+            input: a path of the GenericModule file to unify.
+            output: The output dir to write the unified GenericModule json to.
+            force: if True - Forcefully overwrites the preexisting unified GenericModule file if one exists.
+        """
+
+        if find_type(input) == FileType.GENERIC_MODULE:
+            self.input_path = input
+        else:
+            print_error('You have failed to provide a legal file path, a legal file path '
+                        'should be to a directory of a GenericModule file.')
+
+        self.pack_name = get_pack_name(file_path=self.input_path)
+        self.pack_path = os.path.dirname(os.path.dirname(self.input_path))
+
+        self.input_file_name = os.path.basename(self.input_path).rstrip('.json')
+        self.use_force = force
+
+        if output:
+            if not os.path.isdir(output):
+                print_error('You have failed to provide a legal dir path')
+                sys.exit(1)
+            self.dest_dir = output
+        else:
```

Comment:
```suggestion

            self.dest_dir = output
            
        else:
```
Keep some spaces between if blocks. makes the code easier to read
Same note for all other `if` blocks